uptane/common.py

```diff
@@ -91,15 +91,13 @@ def sign_signable(
 
   # The below was partially modeled after tuf.repository_lib.sign_metadata()
 
-  signatures = []
-
   for signing_key in keys_to_sign_with:
```

Comment:
Two TUF functions that you can look into are sig.verify() and sig.get_signature_status().  They can tell you the number of bad sigs, good sigs, if the signatures satisfy threshold, untrusted sigs, etc.

https://github.com/theupdateframework/tuf/blob/9e63023b30adf764490f9d4b63c1d55960c092ce/tuf/sig.py#L237
https://github.com/theupdateframework/tuf/blob/9e63023b30adf764490f9d4b63c1d55960c092ce/tuf/sig.py#L64